alerts.php

```diff
@@ -234,12 +237,15 @@ function RunAlerts() {
         }
 
         $alert['details'] = json_decode(gzuncompress($alert['details']), true);
+        // noiss - do not issue alert
         $noiss            = false;
+        // noacc - do not close alert ?
         $noacc            = false;
+        // updet - update details in alert_log ?
         $updet            = false;
         $rextra           = json_decode($alert['extra'], true);
         $chk              = dbFetchRow('SELECT alerts.alerted,devices.ignore,devices.disabled FROM alerts,devices WHERE alerts.device_id = ? && devices.device_id = alerts.device_id && alerts.rule_id = ?', array($alert['device_id'], $alert['rule_id']));
-        if ($chk['alerted'] == $alert['state']) {
+        if ($chk['alerted'] == $alert['state']) { // FIXME: this could be made more clear, should probably also be double-checked
```

Comment:
I guess my confusion is on the 'alerted' field in the database.  It sounds like a boolean from the name, is it actually the most recently alerted state?